src/__tests__/core/web_api/p2_api.test.js

```diff
@@ -126,6 +126,14 @@ describe('Auth0APIClient', () => {
         });
         expect(client.authOpt.nonce).toBe('foo');
       });
+      it('should set scope from options.params.scope', () => {
+        const client = getClient({
+          params: {
+            scope: 'foo'
+          }
+        });
+        expect(client.authOpt.scope).toBe('foo');
```

Comment:
This only tests the param was set, can we be sure it made its' way to the authorize URL correctly?

Reply:
added missing test value